### DISCUSSION TITLE
Using gtk::Paned example in relm4?

### DISCUSSION THREAD

**Discussion by cmdcolin** [Relm4/Relm4#608] [Q&A] [answered]
Hi there,
I was looking at ColumnView awhile back (https://github.com/orgs/Relm4/discussions/448), and really glad to see it added. now I was trying to use gtk::Paned but I am a little stuck. I am still quite new to relm4 in general but really wanting to "go beyond" the super imperative gtk style code into the style offered by relm4

I tried something like this

```

    view! {
        gtk::Window {
            set_title: Some("testing"),
            set_default_size: (1200, 600),

            gtk::Paned {
                set_orientation: gtk::Orientation::Vertical,
                set_start_child: Some(&gtk::Button {
                    set_label:"Hello1"
                }),
                set_end_child: Some(&gtk::Button {
                    set_label:"Hello2"
                }),
            }

        }
    }
```

however it says

```
error[E0560]: struct `relm4::gtk4::Button` has no field named `set_label`
   --> src/main.rs:117:13
    |
117 |             set_label:"Hello2"
    |             ^^^^^^^^^ `relm4::gtk4::Button` does not have this field
    |
    = note: all struct fields are already assigned

```

I was confused because there are other examples where gtk::Button uses set_label like this

Thanks!

**Comment by AaronErhardt** [comment, marked as answer]
`=` creates widgets and `:` just assigns variables. As described [here](https://relm4.org/book/stable/component_macro/reference.html#child-widgets), you need the following syntax:

```rust
                #[wrap(Some)]
                set_start_child = &gtk::Button {
                    set_label:"Hello1"
                },
```
